apps/PiGro/uninstall

```diff
@@ -8,8 +8,9 @@ function error {
 }
 
 #if your app installs any packages, keep this command here so those packages will be removed.
-"${DIRECTORY}/purge-installed" "$(dirname "$0")" || exit 1
+"${DIRECTORY}/purge-installed" "neofetch" "$(dirname "$0")" || exit 1
```

Comment:
purge-installed does not work like this.

Reply:
oops